instrumentation/play/play-mvc/play-mvc-2.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_6/Play26Singletons.java

```diff
@@ -66,6 +73,10 @@ public static void updateSpanNames(Context context, Request<?> request) {
     }
   }
 
+  private static void updateHttpRoute(Context context, String route) {
```

Comment:
Nit: this method is so simple that it could be inlined.

Reply:
Yeah, actually this was an intentional thing to try making the aim of the procedure obvious by the method name, but as you mentioned this has already been enough self-descriptive and [ff97ca1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7801/commits/ff97ca15289a0cf8969bc5a02d1c7a9e667d0f08) removed extra lines, so I changed them to be inlined.